components/AsideBlock/AsideBlock.tsx

```diff
@@ -0,0 +1,9 @@
+import { AsideBlockWrapper } from "./style"
+
+const AsideBlock = () => {
+    return (<AsideBlockWrapper>
+
+    </AsideBlockWrapper>)
```

Comment:
should use a single tag

Reply:
soon there will be a new commit where I will add content to these tags